### PR TITLE
Added EmberFest 2021 talks

### PR DESCRIPTION
**Pull request by emmasofiah2o**
Added talks and pictures.

### CODE REVIEW

File: _appearances/2021-09-30-emberfest-2021/appearances/data-validation-in-ember-apps.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+title: 'Data validation libraries in Ember'
+image: '/assets/images/talks/2021-09-30-emberfest-2021/data-validation-in-ember-apps.png'
+url: https://www.youtube.com/watch?v=7rExaNCZRKM
+media: video
+---
+
+Ember Octane’s tracking system makes it easier to integrate libraries from the
+wider JavaScript ecosystem in Ember apps. Bartłomiej Dudzik explained how to
+leverage other packages for validating data than ember-cp-validations or
+ember-changeset at EmberFest 2021.
```
Comment (marcoow) [inline _appearances/2021-09-30-emberfest-2021/appearances/data-validation-in-ember-apps.md]:
```suggestion
ember-changeset.
```

File: _appearances/2021-09-30-emberfest-2021/appearances/making-mira.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+title: 'Making Mira: an Intro to Animating the Web'
+image: '/assets/images/talks/2021-09-30-emberfest-2021/making-mira.png'
+url: https://www.youtube.com/watch?v=QHjN6abvphU
+media: video
+---
+
+Mira is a robot created by Pixar artist Alonso Martinez, capable of recreating a
+great number of emotions and interactions. In this talk from EmberFest 2021,
```
Comment (marcoow) [inline _appearances/2021-09-30-emberfest-2021/appearances/making-mira.md]:
```suggestion
great number of emotions and interactions. In his talk,
```

File: _appearances/2021-09-30-emberfest-2021/appearances/universal-design-systems-the-ember-way.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+title: 'Using Ember to build a universal design system'
+image: '/assets/images/talks/2021-09-30-emberfest-2021/universal-design-systems-the-ember-way.png'
+url: https://www.youtube.com/watch?v=vmspW-tqV8A
+media: video
+---
+
+With Ember being HTML-First, doors open to new possibilities. Chris Manson
+explores the opportunity of building a #designsystem with Ember that can also be
+consumed by apps that are not using JavaScript in this talk from EmberFest 2021.
```
Comment (marcoow) [inline _appearances/2021-09-30-emberfest-2021/appearances/universal-design-systems-the-ember-way.md]:
```suggestion
consumed by apps that are not using JavaScript in his talk.
```

File: _appearances/2021-09-30-emberfest-2021/channel.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+---
+title: 'EmberFest 2021'
+image: '/assets/images/talks/2021-09-30-emberfest-2021/logo.png'
+url: https://emberfest.eu
+---
+
+EmberFest is the European Community Ember Conference and took place in Rome,
+Italy in 2021.
```
Comment (marcoow) [inline _appearances/2021-09-30-emberfest-2021/channel.md]:
```suggestion
EmberFest is the European Community Ember Conference. The 2001 edition took place in Rome,
Italy.
```

**Review by marcoow — COMMENTED** [review]
let's change this a bit so it works better on the website

File: _appearances/2021-09-30-emberfest-2021/channel.md (hunk 4 above)
Comment (emmasofiah2o) [reply _appearances/2021-09-30-emberfest-2021/channel.md]: I guess you meant 2021 :)

Comment (emmasofiah2o) [reply _appearances/2021-09-30-emberfest-2021/channel.md]:
```suggestion
EmberFest is the European Community Ember Conference. The 2021 edition took place in Rome, Italy.
```

File: _appearances/2021-09-30-emberfest-2021/appearances/universal-design-systems-the-ember-way.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+---
+title: 'Using Ember to build a universal design system'
+image: '/assets/images/talks/2021-09-30-emberfest-2021/universal-design-systems-the-ember-way.png'
+url: https://www.youtube.com/watch?v=vmspW-tqV8A
+media: video
+---
+
+With Ember being HTML-First, doors open to new possibilities. Chris Manson
+explores the opportunity of building a #designsystem with Ember that can also be
```
Comment (marcoow) [inline _appearances/2021-09-30-emberfest-2021/appearances/universal-design-systems-the-ember-way.md]:
```suggestion
explores the opportunity of building a design system with Ember that can also be
```

**Review by marcoow — APPROVED** [review]
all good except 2 small things – feel free to merge once fixed

**Pull request merged**